src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -75,6 +78,34 @@ class MinikubeImpl implements Minikube {
     status(): Promise<MinikubeInfo> {
         return minikubeStatus(this.context);
     }
+
+    checkUpgradeAvailable() {
+        return minikubeUpgradeAvailable(this.context);
+    }
+}
+
+async function minikubeUpgradeAvailable(context: Context) {
+    if (!await checkPresent(context, CheckPresentMode.Alert)) {
+        // not installed, no upgrade.
+        return;
+    }
+
+    const sr = await context.shell.exec(`${context.binPath} update-check`);
+    const lines = sr.stdout.split(os.EOL);
+    const currentVersion = extractVersion(lines[0]);
```

Comment:
I might be a bit paranoid but we could check here that `lines.length >= 2` so that if the output is not what we expect we can fail gracefully.  Probably not strictly necessary though.

Reply:
done.